packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -2364,25 +2486,19 @@ class _MenuBarAnchorState extends _MenuAnchorState {
       node: _menuScopeNode,
       skipTraversal: !isOpen,
       canRequestFocus: isOpen,
+      descendantsAreFocusable: true,
```

Comment:
I think this is the default anyhow.  It's not explicitly the default, since it's nullable, but it works out that way. Does setting this actually make a difference?

Reply:
Surprisingly, it did make a difference on my master branch... which I think is clean and not corrupted. The [menus can be traversed multiple times](https://github.com/flutter/flutter/pull/150950/files#diff-4e44be2ef645ef2220e2ad8b83890b9ecb7ed25f71fb90a2cb34ccd4e2f6c263:~:text=%27menus%20can%20be%20traversed%20multiple%20times%27) test fails for me when omitted, but I haven't explored why this is.
```sh
[✓] Flutter (Channel master, 3.24.0-1.0.pre.127, on macOS 14.0 23A344 darwin-arm64, locale en-US)
    • Flutter version 3.24.0-1.0.pre.127 on channel master at /Users/davidhicks/fvm/versions/master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision fbb12c2c61 (3 hours ago), 2024-07-15 18:34:38 -0400
    • Engine revision f1363fb7c2
    • Dart version 3.6.0 (build 3.6.0-36.0.dev)
    • DevTools version 2.37.1
```

Reply:
Well, that is odd. I guess there must be an edge case in there somewhere.